### PR TITLE
Set union added

### PR DESCRIPTION
**Pull request by blowdee**

### CODE REVIEW

**Comment by Gotham13121997** [comment]
@blowdee  Please create a new branch for every pull request you make. Yo will have to reset your master branch to this repos branch. Create a new branch from that master and then add your changes.

**Comment by blowdee** [comment]
@Gotham13121997 Messed up a bit. Now it's merged.

**Comment by Gotham13121997** [comment]
Yeah. By creating new branches you can work on several issues side by side. Now you cant work on another issue unless this one is merged . So take care of it from the next time.

File: underscore.cpp [hunk 1]
```diff
@@ -209,9 +209,73 @@ Collection intersect(Collection collection1, Collection collection2)
...
+	return result;
+}
+
+template <typename Collection, typename ... Collections>
+Collection set_union(Collection collection1, Collection collection2, Collections ... others)
+{
+	return set_union(set_union(collection1, collection2), others...);
+}
+
+	typename Collection::iterator	first_begin = collection1.begin(), second_begin = collection2.begin();
```
Comment (Gotham13121997) [inline underscore.cpp]: This is wrongly placed and will show syntax error

**Comment by Gotham13121997** [comment]
Tested and merged to master

**Comment by Gotham13121997** [comment]
Thanks @blowdee . Please create a new branch if you want to contribute to new issues

**Pull request merged**